deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -391,6 +391,11 @@ spec:
                 - name
                 type: object
               type: array
+            httpPort:
+              description: HttpPort will set the environment env HTTP_PORT to define
```

Comment:
[People, for god's sake, please run `make test` before sending a PR!](https://github.com/kiegroup/kogito-cloud-operator#kogito-operator-collaboration-and-pull-requests)

Reply:
@Kaitou786 do you believe you can do the same thing in the CI check we did for `fmt`, but now on `operator-sdk generate crds|k8s` and `openapi gen`? This way we can warn contributors when subimmiting a PR.

Reply:
@ricardozanini definitely I'll add these checks  :-)